packages/sanity/src/core/releases/components/BundleMenuButton/BundleMenuButton.tsx

```diff
@@ -15,18 +16,32 @@ export const BundleMenuButton = ({bundle}: Props) => {
   const router = useRouter()
   const isBundleArchived = !!bundle?.archivedAt
   const [isPerformingOperation, setIsPerformingOperation] = useState(false)
+  const [showDiscardDialog, setShowDiscardDialog] = useState(false)
+  const [discardStatus, setDiscardStatus] = useState<'idle' | 'discarding' | 'error'>('idle')
 
   const bundleMenuDisabled = !bundle
+  const toast = useToast()
 
   const handleOnDeleteBundle = async () => {
     if (bundleMenuDisabled) return
-
-    setIsPerformingOperation(true)
-    await deleteBundle(bundle._id)
-    setIsPerformingOperation(false)
-    if (router.state.bundleId) {
-      // navigate back to bundle overview
-      router.navigate({bundleId: undefined})
+    try {
+      setDiscardStatus('discarding')
+      await deleteBundle(bundle)
+      setDiscardStatus('idle')
+      if (router.state.bundleId) {
+        // navigate back to bundle overview
+        router.navigate({bundleId: undefined})
```

Comment:
Just to double check: should we use this router navigate vs the sticky params we have so far when setting perspective and such?

Reply:
(I know it was already here before and that's not the change per se, but I just wanted to get a check)

Reply:
This is for the `releases` tool router. It will navigate back to the releases overview screen

Reply:
If you are in the releases detail screen and remove that release, it will send you back to the overview